Bot.py

```diff
@@ -162,10 +159,10 @@ def _postTweet(self, call: dict) -> None:
                         self._cachedTweet = self._api.update_status(msg).id
                     else:
                         self._cachedTweet = self._api.update_status(
-                            msg, self._cachedTweet
+                            msg, in_reply_to_status_id=self._cachedTweet
```

Comment:
This change looks appropriate with the other changes in the commit you referenced!

Reply:
👍🏻